src/test/integration/analytics.test.ts

```diff
@@ -34,4 +34,12 @@ suite('Analytics', () => {
 
     strictEqual(welcomeIsViewed.notCalled, true);
   });
+
+  test('"Welcome Is Viewed" not tracked if using fedramp endpoint', async () => {
+    await configuration.setEndpoint('https://snyk.fedramp.gov');
```

Comment:
I'm getting "Unsafe call of an `any` typed value" trying to run this test 🤔

Reply:
Huh. Just adjusted one of the above. And works on CI and my computer.

Reply:
On which line does your problem occur?

Reply:
I've left the test in so when we tackle HEAD-700, this feature will be tested